doc/api/deprecations.md

```diff
@@ -95,12 +95,18 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
 
-Type: Runtime
+Type: End-of-Life
 
-Within the `cluster` module, the [`worker.suicide`][] property has been
-deprecated. Please use [`worker.exitedAfterDisconnect`][] instead.
+In an earlier version of the Node.js `cluster`, an unfortunate decision was
+made to add a boolean property with the name `suicide` to the `Worker` object.
+The intent of this property was to provide an indication of how and why the
+`Worker` instance exited. The naming of the property is unfortunate not only
```

Comment:
For that, I disagree. One thing we've have consistently needed to get better at is explaining *why* things are deprecated and removed.